src/Shell/Task/BakeTask.php

```diff
@@ -141,9 +141,9 @@ public function callProcess($command)
 
         $error = stream_get_contents($pipes[2]);
         fclose($pipes[2]);
-        proc_close($process);
+        $exit = proc_close($process);
 
-        if ($error) {
+        if ($exit != 0) {
```

Comment:
Right, so the strict check would be best regarding the guideline "be as as strict as possible".